types/generated-snapshot/latest/index.ts

```diff
@@ -523,7 +523,7 @@ export interface DurableObjectId {
   equals(other: DurableObjectId): boolean;
   readonly name?: string;
 }
-export interface DurableObjectNamespace<
+export declare abstract class DurableObjectNamespace<
```

Comment:
Apparently, the fact that `LoopbackDurableObjectNamespace` inherits `DurableObjectNamespace` caused `DurableObjectNamespace` to change from `interface` to `abstract class`.

That's probably harmless but also seems unnecessary -- `LoopbackDurableObjectNamespace` is itself an `interface`, so there's no reason it can't extend another `interface`.

@penalosa Any idea why this happens?